LocalPackages/AIChat/Package.swift

```diff
@@ -0,0 +1,20 @@
+// swift-tools-version: 6.0
```

Comment:
Swift 6 👀

Reply:
Do you see any issues with it? Thought to leave it like that just to be more future-proof, but no strong opinions

Reply:
Nope, actually I'm happy about it since I want us to begin moving to Swift 6 incrementally 🎉